packages/language-server/src/plugins/html/features/astro-attributes.ts

```diff
@@ -55,6 +55,17 @@ export const astroAttributes = newHTMLDataProvider('astro-attributes', {
 						},
 					],
 				},
+				{
```

Comment:
Unrelated to tests, I just noticed that those attributes were missing and felt that it was harmless to do it here